pkg/constants/constants.go

```diff
@@ -14,7 +14,8 @@ const (
 	// DEFAULT_LOGS_COLLECTOR_TIMEOUT is the default timeout for logs collector.
 	DEFAULT_LOGS_COLLECTOR_TIMEOUT = 60 * time.Second
 	// MAX_TIME_TO_WAIT_FOR_POD_DELETION is the maximum time to wait for pod deletion.
-	MAX_TIME_TO_WAIT_FOR_POD_DELETION = 60 * time.Second
+	// 0 seconds for force deletion.
```

Comment:
Before I was thinking that MAX_TIME_TO_WAIT_FOR_POD_DELETION take too long for copy_from_host. Actually, good point, MAX_TIME_TO_WAIT_FOR_POD_DELETION is also for other collectors. So I have revert this change. And add force delete to copy_from_host instead of change polling interval